packages/medusa/src/services/cart.ts

```diff
@@ -1413,35 +1306,34 @@ class CartService extends TransactionBaseService {
    */
   async authorizePayment(
     cartId: string,
-    context: Record<string, unknown> = {}
+    context: Record<string, unknown> & {
+      cart_id: string
+      useExistingTaxLine?: boolean
```

Comment:
suggestion(non-blocking): `snake_case`

Reply:
question: Is this context option used? If not, let's get rid of it

Reply:
I removed the useExisting... but the context is passed to authorizePayment